website/docs/Publishing.md

```diff
@@ -13,6 +13,21 @@ If you are using git and Github pages, you can publish your doc website with one
 
 Otherwise, you can run `moonwave build`, and your built website will be in a folder called `build` in your project directory.
 
+## Using GitHub Actions
+
+When you publish your doc via GitHub Actions you might need to provide the author details and GitHub token.
```

Comment:
You don't actually need to provide your own email and username, you can just provide the GitHub Actions username and email, `github-actions-bot` and `support+actions@github.com`.

Reply:
Yeah maybe you're right. I assumed the same (usually works fine with mock bot address) but got "Failed to get the author data" errors. Maybe it was just me and by default the workflow is just fine with mock author data.